create-custom-map-style/public/index.html

```diff
@@ -6,32 +6,14 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link
-      href="https://api.tiles.mapbox.com/mapbox-gl-js/v2.1.1/mapbox-gl.css"
-      rel="stylesheet"
-    />
     <meta name="theme-color" content="#000000" />
     <link href="%PUBLIC_URL%/site.css" rel="stylesheet" />
+    <link href="https://cdn.maptiler.com/maplibre-gl-js/v1.13.0-rc.4/mapbox-gl.css" rel="stylesheet" />
```

Comment:
Use this instead: `<link href='https://unpkg.com/maplibre-gl@1.15.2/dist/maplibre-gl.css' rel='stylesheet' />`

Newer version, vendor-neutral CDN.